linkerd/examples/k8s.yaml

```diff
@@ -0,0 +1,19 @@
+admin:
+  port: 9990
+
+namers:
+- kind: io.l5d.experimental.k8s
+  host: localhost
+  port: 443
+  tls: true
+  tlsWithoutValidation: true
+  authTokenFile: linkerd/examples/io.l5d.k8s/kube_token
```

Comment:
nothing inherently wrong with that
It's not tested though (acceptance-test.yaml is), so it could come back to bite us as these examples fall out of date